javascript-bank/account.js

```diff
@@ -0,0 +1,43 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  var positive = false;
+  var depositObject = new Transaction();
+  if (amount > 0 && amount % 1 === 0) {
+    positive = true;
+    depositObject.type = 'deposit';
+    depositObject.amount = amount;
+    this.transactions.push(depositObject);
+  }
+  return positive;
+};
+
+Account.prototype.withdraw = function (amount) {
+  var positive = false;
+  var withdrawObject = new Transaction();
```

Comment:
Same deal here, make it when you know you have the right data and pass that data as arguments